### PR TITLE
Remove old testnet config

### PR DESCRIPTION
**Pull request by zuzunker**

### CODE REVIEW

File: exe-common/src/config.rs [hunk 1]
```diff
@@ -316,37 +316,17 @@ pub mod net {
...
-                )
-                .unwrap(),
-                epoch_start: 0,
-                epoch_stability_depth: DEFAULT_EPOCH_STABILITY_DEPTH,
-                protocol_magic: ProtocolMagic::from(1097911063),
-                peers: peers,
-            }
-        }
-
+        /*
```
Comment (rooooooooob) [inline exe-common/src/config.rs:319]: Small typo at start of comment: This config is *named*

**Review by rooooooooob — APPROVED** [review]
LGTM. The only minor issue I see is a pretty insignificant typo in the comment about testnet2

File: exe-common/src/config.rs (hunk 1 above)
Comment (zuzunker) [reply exe-common/src/config.rs:319]: 🙌

File: exe-common/src/config.rs [hunk 2]
```diff
@@ -316,37 +316,17 @@ pub mod net {
...
+        This config is names `testnet2` because it's a new rebooted alternative testnet chain
```
Comment (zuzunker) [inline exe-common/src/config.rs]:
```suggestion
        This config is named `testnet2` because it's a new rebooted alternative testnet chain
```

**Pull request merged**